ckanext/cnra_schema/plugins.py

```diff
@@ -26,6 +29,21 @@ def update_config(self, config):
 """
 
     def get_package_dict(self, context, data_dict):
+        harvest_object = data_dict['harvest_object']
+        harvest_source_type = harvest_object.source.type
+
+        if harvest_source_type == 'fgdc':
```

Comment:
I've tested the CSW FGDC harvester and the source type should be set to `csw_fgdc`.
https://github.com/OpenGov-OpenData/ckanext-spatial/blob/master/ckanext/spatial/harvesters/csw_fgdc.py#L94

Reply:
You're right, I just finished testing and confirmed that the source type needs to be `csw_fgdc`. I have pushed this change